packages/estransform/lib/parcel-source-map-default.cjs

```diff
@@ -0,0 +1,3 @@
+// we need to require the module for compatibility with the default export
+// eslint-disable-next-line @typescript-eslint/no-var-requires
+module.exports = require('@parcel/source-map').default;
```

Comment:
this workaround is suggested in https://github.com/evanw/esbuild/issues/532
and it's similar to another one you use in `packages/rna-storybook/lib/csf-tools.cjs`